course/assignments/reproducible-research-assignment.md

```diff
@@ -0,0 +1,21 @@
+## Reproducible Research Plan:
+
+Based on the [Data Reuse Plan](https://github.com/mozillascience/working-open-workshop/blob/gh-pages/handouts/data_reuse_plan_template.md) by Mozilla WOW.  
+
+Create a README.Rmd that describes the data or software package. It can be pretty sparse for now but you will populate it as you create your project. It must have the following sections (If a section does not apply to your project, remove it and when you submit your assignment, include a Notes.txt explaining why it is not applicable)
```

Comment:
For the file naming, it might be best to encourage them to follow a standard that has been developed, rather than create their own. Like Google's R style guide https://google.github.io/styleguide/Rguide.xml